src/validations/activity.js

```diff
@@ -3,8 +3,8 @@ const Joi = require('joi');
 
 const validateCreation = (req, res, next) => {
   const activityValidation = Joi.object({
-    name: Joi.string().min(4).pattern(/^[a-zA-Z]+$/).required(),
-    description: Joi.string().min(5).required(),
+    name: Joi.string().min(4).max(20).required(),
```

Comment:
i dont get why you remove the regex (pattern in this case) from validateCreation but then keep it on validateUpdateActivity. In this case when you create an activity, name could contain numbers and different symbols but when you update an activity only letters without any spaces.  You should decide if keep it on both validation or remove it.